rfcs/0008-pablo-lbp-cpp-restructure.md

```diff
@@ -3,74 +3,45 @@ Design Proposal: Pablo Formalization and Restructure
 
 Table of Contents
```

Comment:
Honestly? IMO just drop the table of contents.

Reply:
I just started to remove some noise from useless spec to retain only usefull pieces. all that thing should have been in docs or project item, not in RFC.